src/components/layout/AddToWishlistButton/AddToWishlistButton.js

```diff
@@ -4,49 +4,39 @@ import axiosInstance from '../../../apis/axiosClient'; // axios instance
 import { AuthStatus, useAuth } from '@akosasante/react-auth-context';
 import { useNavigate } from 'react-router-dom';
 
-const wishlistEndpoint = '/wishlist';
```

Comment:
This deleted line is making the api calls for addToWishlist and removeFromWishlist erorr out currently. Either restore this line or update those api calls so that they just hardcode the string `/wishlist`

Reply:
Sorry, I accidently removed that line after I checked that addToWishlist and removeFromWishlist works good, I restored taht